examples/pthreads/svcomp2015/unseen/scull_safe.h

```diff
@@ -0,0 +1,72 @@
+#define inode int // the lock
```

Comment:
I put all of the benchmarks I wanted to evaluate into this unseen folder, and then as I assessed them for suitability for benchmarking, I moved them into the other folders.

I don't remember why this scull example never got moved into one of the other folders.  As I mentioned, I'll go through all the examples so I know why each of them is categorized as they are.